torchmultimodal/modules/encoders/albef_text_encoder.py

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import math
+from typing import Optional
+
+import torch
+from torch import nn, Tensor
+from torchmultimodal.utils.common import get_extended_attention_mask
+
+
+class ALBEFTextEncoder(nn.Module):
+    """
+    Construct word embeddings from input_ids and attention_mask.
+
+    Based on https://github.com/huggingface/transformers/blob/main/src/transformers/models/bert/modeling_bert.py#L870
+
+    Args:
+        vocab_size (int): Vocabulary size of the model. Defines the different tokens that can be represented by the inputs_ids.
+        hidden_size (int): Dimensionality of the encoder layers.
+        num_hidden_layers (int): Number of hidden layers in the Transformer encoder.
+        num_attention_heads (int): Number of attention heads for each attention layer in the Transformer encoder.
+        intermediate_size (int): Dimensionality of the “intermediate” (i.e., feed-forward) layer in the Transformer encoder.
+        max_position_embeddings (int): The maximum sequence length that this model might ever be used with.
+        type_vocab_size (int): The vocabulary size of the token_type_ids.
+        pad_token_id (int): The embedding for pad_token_id is not updated during training.
+        layer_norm_eps (float): The epsilon used by the layer normalization layers.
```

Comment:
nit: add `Default is ___.` to the ends of these lines since there are default values. docstring convention :)